spring-cloud-aws-starters/spring-cloud-aws-starter-s3-config/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need a dedicated starter for it? or should just s3 started be used?

Reply:
I think this is bug in IntelliJ I don't see this file